knative-operator/deploy/resources/monitoring/rbac-proxy-roles.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Should we make this part of the serving manifest?

Reply:
Which manifest?